inference_segmentation.py

```diff
@@ -1,46 +1,45 @@
-import gc
-import logging
-import warnings
 from math import sqrt
```

Comment:
Strangely, the previous inference.py in this diff is not the last version of inference.py in NRCan/develop: https://github.com/NRCan/geo-deep-learning/blob/develop/inference.py

This may be a bug with git. We'll need to address this as many changes have been brought by Vic in the last inference.py (ex.: swtiching to a generator function when reading the image to infer on)